src/views/_proxied-dot-io/product-docs-landing/partials/featured-card/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactElement } from 'react'
```

Comment:
nit: what do you think about renaming this directory to `components` from `partials`?

Reply:
Would be on board with that, didn't make a comment on it since this is in `_prodixed-dot-io` and we have different standards in each part of the app.